### DISCUSSION TITLE
GenerateReleaseNotes test console 401 when getting builds

### DISCUSSION THREAD

**Discussion by Smiffc0** [rfennell/AzurePipelines#1106] [Q&A] [answered]
Hi. I am running the test console against an Azure project and am having an issue I hope you can help with.

I am using a PAT token and several other API calls get processed without issue but when the call to `https://{azure_tenant}/{project}/_apis/build/Builds/{buildId}` executes it always returns a `401 Unauthorised`. error.

Stepping through I can see that the url is correct and it opens correctly if I paste it into a browser, but the console throws a 401 without fail.

Any help would be most appreciated

**Comment by rfennell** [comment, marked as answer]
A couple of things to try

- In the test console JSON settings file, it is point to the correct Azure DevOps organisation and team project?
- If you use [Postman](https://www.postman.com/) can you make a call to the REST API with that PAT (it could the that PAT has n build access). A browser test won't use the PAT

**Comment by Smiffc0** [comment]
That was indeed the case. I gave the PAT more permissions and the test console is now running!
Thanks for the swift reply
